.manylinux-install.sh

```diff
@@ -11,6 +11,12 @@ for PYBIN in /opt/python/*/bin; do
        [[ "${PYBIN}" == *"cp38"* ]]; then
         "${PYBIN}/pip" install -e /io/
         "${PYBIN}/pip" wheel /io/ -w wheelhouse/
+        if [ `uname -m` == 'aarch64' ]; then
+         cd /io/
+         "${PYBIN}/pip" install tox
+         "${PYBIN}/python" -m tox -e py`echo "${PYBIN}" | cut -f 4 -d"/" | cut -f 1 -d"-" | cut -c3-`
```

Comment:
This could be simplified to 

```suggestion
         "${PYBIN}/tox" -e py
```

Reply:
@mgedmin  Thank you for the feedback.
I have done the changes as suggested by you. Please check.